blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionHandler.java

```diff
@@ -124,8 +124,8 @@ public void handle(ChannelHandlerContext ctx, HttpRequest request) {
                 containerMetrics = jsonMetricsContainer.toMetrics();
                 forceTTLsIfConfigured(containerMetrics);
 
-                if (!jsonMetricsContainer.areDelayedMetricsPresent()) {
-                    Tracker.trackDelayedMetricsTenant(tenantId);
+                if (jsonMetricsContainer.areDelayedMetricsPresent()) {
```

Comment:
why is the logic swapped around from not to is present?  did we get it wrong initially?